stubs/invoke/invoke/context.pyi

```diff
@@ -1,15 +1,16 @@
 from contextlib import AbstractContextManager
 
 from .config import Config, DataProxy
+from .runners import Result
 
 class Context(DataProxy):
     def __init__(self, config: Config | None = ...) -> None: ...
     @property
     def config(self) -> Config: ...
     @config.setter
     def config(self, value: Config) -> None: ...
-    def run(self, command: str, **kwargs): ...
-    def sudo(self, command: str, *, password: str = ..., user: str = ..., **kwargs): ...
```

Comment:
Looks like these two kw-only params actually exist here: https://github.com/pyinvoke/invoke/blob/main/invoke/context.py#L179-L180

No need to remove them.